patches/node/build_add_gn_build_files.patch

```diff
@@ -70,15 +74,11 @@ index 0000000000000000000000000000000000000000..446119163d1f7bad577cb0b7b217ecf2
 +
 +copy("node_js2c_inputs") {
 +  sources = library_files
-+  outputs = [
-+    "$target_gen_dir/js2c_inputs/{{source_target_relative}}",
-+  ]
++  outputs = [ "$target_gen_dir/js2c_inputs/{{source_target_relative}}" ]
```

Comment:
would it be possible to do the `gn format` in a separate commit? it's hard to see what changed here.

Reply:
Not now, it's too late I think, I'd have to undo the format manually :rip:

Reply:
can you reformat it as a separate PR, commit that, then merge that in here?

Reply:
Ah hm, I suspect that will result in some nasty merge conflicts.  Let me try locally

Reply:
@nornagon The isolated patch file updates are in this commit --> https://github.com/electron/electron/pull/27113/commits/a795072feb8644de73281b8bb21d413273018cf2